pkg/runtime/framework/plugins/plainml/plainml.go

```diff
@@ -58,7 +58,7 @@ func (p *PlainML) EnforceMLPolicy(info *runtime.Info, trainJob *trainer.TrainJob
 
```

Comment:
Are we planning to change other plugins in the followup PRs ?
E.g. insert NumNodes to the info.RuntimePolicy.MLPolicy.NumNodes ?

Reply:
As we discussed in previously, I prioritized this MPI enhancement instead of stabilize plugin mechanism.
So, the whole of refactor will be performed in the future PRs as part of https://github.com/kubeflow/trainer/issues/2495.

Reply:
Thanks!